de.js

```diff
@@ -0,0 +1,334 @@
+    wallet_address: `Wallet-Adresse`,
+    wallet_label: `Wallet-Bezeichnung`,
+    save_changes: `Änderungen speichern`,
+    save_address: `Adresse speichern`,
+    enter_new_PIN: `Neue PIN eingeben`,
+    confirm_new_pin: `Neue PIN bestätigen`,
+    enter_new_PIN_description: `Gib deine neue PIN für deine MyDoge-Wallet ein`,
+    enter_old_PIN: `Alte PIN eingeben`,
+    enter_old_PIN_description: `Gib deine alte PIN für deine MyDoge-Wallet ein`,
+    forgot_PIN: `PIN vergessen`,
+    confirm: `Bestätigen`,
+    biometrics_confirmation: `Möchtest du deine Wallet mithilfe von Biometrie entsperren?`,
+    use_biometrics: `Biometrie benutzen`,
+    dismiss: `Ablehnen`,
+    remove_wallet_data_description: ` Dies wird alle Wallet-Daten von deinem Gerät entfernen`,
+    delete_profile_forever: `MyDoge-Profil für immer löschen`,
+    sign_out: `Abmelden`,
+    delete_wallet_confirmation: `Hey! Bist du sicher?`,
+    remove_wallet_description: `Deine jetzige Wallet wird von der App entfernt, sie wird jedoch für immer auf der Blockchain existieren.`,
+    remove_profile_assets_description: `Dein Benutzerprofil und deine Uploads werden entfernt und dauerhaft gelöscht. Diese Aktion kann nicht rückgängig gemacht werden.`,
+    wallet_recovery_description: ` Die Wiederherstellung dieser Wallet ist NUR möglich mit deiner`,
+    seed_phrase: `Seed-Phrase`,
+    no_seed_phrase: `MyDoge ist nicht im Besitz deiner Seed-Phrase.`,
+    understand_continue: `Verstanden. Weiter`,
+    enter: `Tippe`,
+    Delete: `Löschen`,
+    erase_wallet: `zum Löschen dieser Wallet`,
+    and_profile_data: `und deiner Profil-Daten`,
+    message: `Nachricht`,
+    username: `Benutzername`,
+    edit: `Bearbeiten`,
+    new_address: `Neue Adresse`,
+    your_seed_phrase: `Deine Seed-Phrase ✍️`,
+    save_seed_phrase_instructions: ` Sichere diese Wörter in der richtigen Reihenfolge. Schreibe sie auf und verwahre sie an einem sicheren Ort oder kopiere sie in einen Passwort-Manager.`,
+    seed_phrase_warning: `Deine Seed-Phrase ist der Schlüssel zu deiner Wallet. Jeder mit dieser Wortfolge kann auf dein Guthaben zugreifen. Tippe hier um mehr zu erfahren.`,
+    copy: `Kopieren`,
+    hide: `Verstecken`,
+    show: `Zeigen`,
+    screenshot_detected: `Screenshot erkannt`,
+    screenshot_warning: `Screenshots oder andere digitale Kopien deiner Seed-Phrase könnten in fremde Hände gelangen!`,
+    okay: `Okay`,
+    seed_phrase_importance: `Wie wichtig ist meine Seed-Phrase?`,
+    seed_phrase_explanation: `Jeder mit deiner Seed-Phrase kann auf dein Guthaben zugreifen. Teile deine Seed-Phrase mit niemandem. MyDoge speichert deine Seed-Phrase nicht für dich. Deine Seed-Phrase ist die einzige Möglichkeit deine Wallet wiederherzustellen solltest du dein Gerät jemals verieren, es beschädigt oder gestohlen werden, oder in sonstiger Form kein Zugriff mehr darauf bestehen.`,
+    send_receive_tips_x: `Tips per 𝕏 senden und empfangen`,
+    read_mydogetip_faq: `Lies die MyDogeTip-FAQ`,
```

Comment:
Since 'MyDogeTip FAQ' seems to be a proper name there's no need to change it yet... but one could write 'MyDoge-Trinkgeld-FAQ' maybe the website gets a german translation somewhen too (I'd help there too, if you like :) 

```suggestion
    read_mydogetip_faq: `Lies die MyDoge-Trinkgeld-FAQ`,
```

Reply:
Will leave as MyDogeTip for consistency since it's @MyDogeTip on X